app/scripts/modules/kubernetes/serverGroup/configure/configure.kubernetes.module.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.serverGroup.configure.kubernetes', [
+  require('../../../core/account/account.module.js'),
+  require('./Configuration.service.js'),
```

Comment:
We capitalize controllers, but not services - can you change this file name to `configuration.service.js`? I hate to ask because I know what a pain it can be with git to just change the case of a file name, but it'll be a good exercise for you!